doc/components.md

```diff
@@ -59,8 +59,12 @@ Example of usage, where visibility of two components are handled by a wrapper:
 ```dart
 class GameOverPanel extends PositionComponent with HasGameRef<MyGame> {
   bool visible = false;
+  final Image spriteImage;
 
-  GameOverPanel(Image spriteImage) : super() {
+  GameOverPanel(this.spriteImage) : super();
```

Comment:
no need to call empty super right

Reply:
super is always called, if not specified the default super constructor is called (AFAIK)